docs/index.asciidoc

```diff
@@ -25,6 +25,34 @@ This Logstash input plugin allows you to call an HTTP API, decode the output of
 send them on their merry way. The idea behind this plugins came from a need to read springboot
 metrics endpoint, instead of configuring jmx to monitor my java application memory/gc/ etc.
 
+[id="plugins-{type}s-{plugin}-ecs_metadata"]
+==== Event Metadata and the Elastic Common Schema (ECS)
+
+This input will add metadata about the HTTP connection itself to each event.
+
+When ECS compatibility is disabled, metadata was added to a variety of non-standard top-level fields, which has the potential to create confusion and schema conflicts downstream.
+
+With ECS Compatibility Mode, we can ensure a pipeline maintains access to this metadata throughout the event's lifecycle without polluting the top-level namespace.
+
+Here’s how ECS compatibility mode affects output.
+[cols="<l,<l,e,<e"]
+|=======================================================================
+| ECS disabled | ECS v1 | Availability | Description
+
+| [@metadata][request][url] | [url][full] | Always | The URL of the endpoint
+| [@metadata][request][method] | [http][request][method] | Always | HTTP request method
+| [@metadata][host] | [host][hostname]  | Always | Hostname
+| [@metadata][code] | [http][response][status][code] | When server responds a valid status code | HTTP response code
+| [@metadata][response_headers] | [@metadata][input][http_poller][response][headers] | When server responds with headers | HTTP headers of the response
+| [@metadata][response_message] | [@metadata][input][http_poller][response][status][message] | When server responds with status line | message of status line of HTTP headers
+| [@metadata][runtime_seconds] / [http_request_failure][runtime_seconds] | [@metadata][input][http_poller][response][time][second] | Always | response time of calling endpoint
+| [@metadata][times_retried] | [@metadata][input][http_poller][request][retried] | When the poller calls server successfully | retry count from manticore
```

Comment:
> retry count from manticore

Manticore is an internal implementation detail, and adding it to our docs ties us to its implementation. It is better for us to be a little vague here.

```suggestion
| [@metadata][times_retried] | [@metadata][input][http_poller][request][retried] | When the poller calls server successfully | retry count from http client library
```